extensions/wikidata/module/langs/translation-en.json

```diff
@@ -126,6 +126,10 @@
     "warnings-messages/no-edit-generated/body": "There might be something wrong with your schema.",
     "warnings-messages/no-issue-detected/title": "No issue was detected in your edits.",
     "warnings-messages/no-issue-detected/body": "Note that OpenRefine cannot detect all the types of problems Wikidata edits can have.",
+    "warnings-messages/multi-valued-property-is-required-for-new-item/title": "{property_entity} should have more than one statements.",
+    "warnings-messages/multi-valued-property-is-required-for-new-item/body": "This property is expected to have more than one statement on each item but it has single statement, for instance on {example_entity}.",
+    "warnings-messages/multi-valued-property-is-required-for-existing-item/title": "{property_entity} should have more than one statements.",
+    "warnings-messages/multi-valued-property-is-required-for-existing-item/body": "This property is expected to have more than one statement on each item but it has single statement, for instance on {example_entity}. It could be a false warning that would be removed once the item is uploaded to wikidata as it might have additional values over there already.",
```

Comment:
In the second sentence, capital letter for "Wikidata". I would also rephrase the sentence to something like `If the item already has statements with this property in Wikidata, then this warning can be ignored.`